src/util/apiUtils.js

```diff
@@ -26,6 +27,13 @@ const MOCK_RESPONSE_OK = {  // minimal representation of http.IncomingMessage
 	},
 };
 
+export const createCookieJar = apiUrl => {
+	if (url.parse(apiUrl).pathname === '/sessions') {
```

Comment:
nitpick, but can this check was moved to where the method is being invoked in `buildRequestArgs`. Seems more a part of that business logic.